.readthedocs.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+
+sphinx:
+  # This should not fail on errors so that we can view the rendered Read The Docs
+  # pages even on errors.
+  # We have a separate build process which fails on errors and shows all
+  # the errors in one go.
+  # This can be re-enable once RTD is updated with
+  # https://github.com/readthedocs/readthedocs.org/issues/7116
+  fail_on_warning: false
+
+formats:
+  - pdf
+
+
+python:
+  version: 3.8
```

Comment:
Why not 3.9?

Reply:
3.9 will work... I remember seeing 3.8 forces as the python version for building the docs inside tox... so I went with that version.